spec/lib/pdf_filler/nj1040_pdf_spec.rb

```diff
@@ -1613,6 +1613,50 @@
       end
     end
 
+    describe "line 55 - total income tax withheld" do
+      context 'when TaxWithheld has a value' do
+        let(:intake) { create(:state_file_nj_intake, :df_data_many_w2s) }
```

Comment:
[pebble] sorry to keep harping on this but i do think it's really useful to mock the calculator method. that way if the fixture changes it doesn't break a bunch of specs beyond just the calculator spec